### ISSUE TITLE
Make language picker focusable 

### ISSUE THREAD

**Issue by jasonnrosa** [HubSpot/cms-theme-boilerplate#214]
Currently the language picker on the boilerplate website is not focusable meaning that a user navigating by keyboard can't access it like they can the main menu. We should repurpose the menu JavaScript to ensure that the language picker is keyboard accessible (original suggestion here: https://github.com/HubSpot/cms-theme-boilerplate/issues/206). 

**Checklist**

- [X] I have read the [CONTRIBUTING](https://github.com/HubSpot/cms-theme-boilerplate/blob/master/CONTRIBUTING.md) document.
- [X] I have checked the [issue tracker](https://github.com/HubSpot/cms-theme-boilerplate/issues) to make sure there isn’t already an open issue.

**Comment by jasonnrosa** [comment]
Going to close this out for now. This is certainly something that we want to address but it is not something that is controlled within boilerplate. We'll need to make an update to the language switcher module so that it is accessible using the keyboard.
